src/interfaces/IParam.sol

```diff
@@ -2,40 +2,41 @@
 pragma solidity ^0.8.0;
 
 interface IParam {
+    /// @notice LogicBatch is signed by a signer using EIP-712
     struct LogicBatch {
-        Logic[] logics;
-        Fee[] fees;
-        uint256 deadline;
+        Logic[] logics; // An array of logics to be executed
```

Comment:
Feel like this way is better, [ref](https://github.com/aave/aave-v3-core/blob/29ff9b9f89af7cd8255231bc5faf26c3ce0fb7ce/contracts/deployments/ReservesSetupHelper.sol#L31).  What do you think ?

```suggestion
        Logic[] logics; // An array of Logic struct to be executed
```

Reply:
Use `Logic`, `Input` and `Fee` structs in comment, thanks.

Reply:
I think we should use plurals after `an array of`. What do you think?

Reply:
What's your reason ?  I thought it means: an array of `xxx struct`.